specifyweb/specify/test_trees.py

```diff
@@ -0,0 +1,215 @@
+from specifyweb.specify import models
+from specifyweb.specify.api_tests import ApiTests
+from specifyweb.specify.tree_stats import get_tree_stats
+from specifyweb.stored_queries.tests import SQLAlchemySetup
+
+
+def get_table(name: str):
```

Comment:
(optional) 
This is personal preference, but I would move this to a more 'general' file. 
Perhaps to `tests.py` if you wish to keep it maintained in the "testing" environment. 
This function may be helpful when writing other (WorkBench) tests, and I would not expect such a function in a `test_trees.py` file. 

I was surprised to find I could not find a similar function in the "main" non-test files (there is the `get_table` and `get_table_strict` on the datamodel object, but seemingly no equivalent for Django). 
I know in most places we are either directly accessing the models from the model object (`models.Collectionobject`), or using something like the implementation here via getattr. 
We could perhaps use this in those places as well, and handle things such as AttributeErrors better (which are not being handled at all currently).

Reply:
I was surprised too! AttributeErrors are actually fine to not handle. If you want to perform an action with a table (that's what a model just a representation of) but that table doesn't exist, you can't do anything. So, just propagate that error up is a good choice.